### PR TITLE
Hello page

### PR DESCRIPTION
**Pull request by letarulinda**
Created a hello page with a Hello Message.
Created a gitignore file to prevent tracking the test.md file
Created a test.md file
Modified the README Template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,18 @@
...
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
```
Comment (Kossi-stack) [inline README.md]: - [ ] Kindly remove this text.

File: README.md [hunk 2]
```diff
@@ -1,72 +1,18 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
 # Project Name
 
-> Description the project.
```
Comment (Kossi-stack) [inline README.md:5]: - [ ] Kindly add a short description of the project.

File: README.md [hunk 3]
```diff
@@ -1,72 +1,18 @@
...
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
```
Comment (Kossi-stack) [inline README.md:54]: - [ ] Kindly remove Author 2 since you work alone and make sure you provide the correct socials handle.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
**Hi @letarulinda ,**

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

**To Highlight**

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Commit messages is meaningful.

**Required Changes ♻️**

 - [ ] Kindly add node_modules to .gitignore file.
 - [ ] Please make sure to use [this readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and please be sure to edit all of the sections to better suit your project.

 _Check the comments under the review for more changes_.

**Optional suggestions**

 - [ ] Kindly edit the PR's title to something more descriptive. It can be "Hello Microverse Project PR or Hello Microverse PR".
 
  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by koachcode — APPROVED** [review]
Hi @letarulinda , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**